CoffeeApp.js

```diff
@@ -0,0 +1,98 @@
+﻿"use strict";
+var LintStream = require('jslint').LintStream;
+var fs = require('fs');
+var csv = require('fast-csv');
+var obj = csv();
+
+var readline = require('readline');
+var rdl = readline.createInterface(process.stdin, process.stdout);
+
+var longi;
+var lati;
+var file;
+var latitude = [];
+var longitude = [];
+var location = [];
+var ct = 0;
+
+rdl.question("Datele: ", function (data) {
+    var elementss = data.split(','),
+        value,
+        lat1,
+        lon1,
+        lat2,
+        lon2;
+    longi = elementss[0];
+    lati = elementss[1];
+    file = elementss[2];
+
+    function interpretation(value) {
+        location = [...location, `${value[0]}`];
+        latitude = [...latitude, `${value[1]}`];
+        longitude = [...longitude, `${value[2]}`];
+    }
+
+    function deg2rad(deg) {
+        return deg * (Math.PI / 180);
+    }
+
+    function getDistance(lat1, lon1, lat2, lon2) {
+        var R = 6371,// Radius of the earth in km

```

Comment:
This is most likely correct,

But the readme states:

```
Assume all coordinates lie on a plane.
```

So this makes for overengineering, can you tell me why this is considered usually a bad practice?

Reply:
Overengineering is considered a bad practice because you make a solution more complicated than it needs to be to fit your needs